MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1807,13 +1807,13 @@ public string GetVideoQualityParam(EncodingJobInfo state, string videoEncoder, E
             }
             else if (hardwareAccelerationType == HardwareAccelerationType.qsv)
             {
-                if (OperatingSystem.IsLinux())
+                if (OperatingSystem.IsLinux() || OperatingSystem.IsFreeBSD())
                 {
                     var ver = Environment.OSVersion.Version;
                     var isFixedKernel60 = ver.Major == 6 && ver.Minor == 0 && ver >= _minFixedKernel60i915Hang;
                     var isUnaffectedKernel = ver < _minKerneli915Hang || ver > _maxKerneli915Hang;
 
-                    if (!(isUnaffectedKernel || isFixedKernel60))
+                    if (!(isUnaffectedKernel || isFixedKernel60 || OperatingSystem.IsFreeBSD()))
```

Comment:
can you explain this change please?

Reply:
my understanding of this part is this is workarounding an issue with the linux kernel driver at some point concerning the drm drivers, the FreeBSD version of the drm drivers are the drm drivers from 6.1-LTS so they should not be affected